assets/app/scripts/directives/resources.js

```diff
@@ -18,7 +18,15 @@ angular.module('openshiftConsole')
         // Pods
         pods: '='
       },
-      templateUrl: 'views/_overview-deployment.html'
+      templateUrl: 'views/_overview-deployment.html',
+      controller: function($scope) {
+        $scope.viewPodsForDeployment = function(deployment) {
+          $location.url("/project/" + deployment.metadata.namespace + "/browse/pods");
+          $timeout(function() {
+            LabelFilter.addActiveFilter("deployment", "in", [deployment.metadata.name]);
```

Comment:
Hmm I don't think we have a guaranteed label for the pods of a regular RC, probably need to not show the link for those. Will double check tmrw.